performance_regression_alternator_test.py

```diff
@@ -0,0 +1,211 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2020 ScyllaDB
+
+from performance_regression_test import PerformanceRegressionTest
+from sdcm.utils.alternator import create_table as alternator_create_table, set_table_write_isolation
+from sdcm.utils.common import normalize_ipv6_url
+
+
+class PerformanceRegressionAlternatorTest(PerformanceRegressionTest):
+    def _workload(self, stress_cmd, stress_num, test_name=None, sub_type=None, keyspace_num=1, prefix='', debug_message='',  # pylint: disable=too-many-arguments,arguments-differ
+                  save_stats=True, check_regression=False):
+        if debug_message:
+            self.log.debug(debug_message)
+
+        if save_stats:
+            self.create_test_stats(test_name=test_name, sub_type=sub_type,
+                                   doc_id_with_timestamp=True, append_sub_test_to_name=False)
+        stress_queue = self.run_stress_thread(stress_cmd=stress_cmd, stress_num=stress_num, keyspace_num=keyspace_num,
+                                              prefix=prefix, stats_aggregate_cmds=False)
+        self.get_stress_results(queue=stress_queue, store_results=True)
+        if save_stats:
+            self.update_test_details(scylla_conf=True)
+            # self.display_results(results, test_name=test_name)
+            if check_regression:
+                self.check_regression()
```

Comment:
May remove this code, and appropriate parameter from signature? because it is not used in code

Reply:
fixed